packages/libglibutil-deb/libglibutil-deb.pacscript

```diff
@@ -0,0 +1,14 @@
+pkgname="libglibutil-deb"
+gives="libglibutil"
+replaces=("${gives}")
+arch=("amd64" "arm64")
+repology=("project: ${gives}")
+pkgver=("1.0.66")
+maintainer=("Slava Monich <slava.monich@jolla.com>" "James Ed Randson <jimedrand@disroot.org>")
```

Comment:
Who is Slava Monich?

Reply:
According to what I found on Bing, he's/she's (I don't know exactly what his/her pronouns cause I cannot found his gender info) senior Software Engineer at TGI Sport. I don't know so much about him/her, but I found the maintainer info from sources I got

Sources/references : 
https://repo.waydro.id/dists/noble/main/binary-all/Packages (I found maintainer infos here)
https://github.com/monich (His/her GitHub)
https://revonect.com/contact/Slava%20Monich/id/c9d47030111 (His/her ID on Revonict)

That's all I got from the web about Slava Monich.

Reply:
`maintainer` is specifically for the maintainers of the Pacstall package. Unless this person has personally agreed to maintain this pacscript, they should not be included here

Reply:
> `maintainer` is specifically for the maintainers of the Pacstall package. Unless this person has personally agreed to maintain this pacscript, they should not be included here

So, I should remove him/her ? Even I add it as credits ?